common/src/x509.rs

```diff
@@ -0,0 +1,79 @@
+// Copyright 2021 Oxide Computer Company
```

Comment:
I probably forgot this in my PR.. need to go check.
New year, 2022!

Reply:
Addressed in 274002b :)